infra/src/test/kotlin/fi/fta/geoviite/infra/TestDBService.kt

```diff
@@ -0,0 +1,379 @@
+                "alignment",
+                "alignment_version",
+                "km_post",
+                "km_post_version",
+                "location_track",
+                "location_track_version",
+                "reference_line",
+                "reference_line_version",
+                "switch",
+                "switch_version",
+                "switch_joint",
+                "switch_joint_version",
+                "track_number",
+                "track_number_version",
+                "segment_version",
+                "segment_geometry",
+            ),
+        )
+    }
+
+    fun clearGeometryTables() {
+        deleteFromTables(
+            schema = "geometry",
+            tables = arrayOf(
+                "alignment",
+                "cant_point",
+                "element",
+                "plan",
+                "plan_application",
+                "plan_application_version",
+                "plan_file",
+                "plan_project",
+                "plan_project_version",
+                "plan_author",
+                "plan_author_version",
+                "plan_version",
+                "switch",
+                "switch_joint",
+                "vertical_intersection",
+            )
+        )
+    }
+
+    fun clearPublicationTables() {
+        deleteFromTables(
+            schema = "publication",
+            tables = arrayOf(
+                "km_post",
+                "location_track",
+                "location_track_km",
+                "publication",
+                "reference_line",
+                "switch",
+                "switch_joint",
+                "switch_location_tracks",
+                "track_number",
+                "track_number_km",
+            ),
+        )
+    }
+
+    fun clearSplitTables() {
+        deleteFromTables(
+            schema = "publication",
+            tables = arrayOf(
+                "split",
+                "split_version",
+                "split_relinked_switch",
+                "split_relinked_switch_version",
+                "split_target_location_track",
+                "split_Target_location_track_version",
+            ),
+        )
+    }
+
+    fun clearProjektivelhoTables() {
+        deleteFromTables(
+            schema = "projektivelho",
+            tables = arrayOf(
+                "document_content",
+                "document_rejection",
+                "document",
+                "document_type",
+                "material_category",
+                "material_group",
+                "material_state",
+                "technics_field",
+                "project_state",
+                "assignment",
+                "project",
+                "project_group",
+                "search",
+            ),
+        )
+    }
+
+    fun clearRatkoTables() {
+        deleteFromTables(
+            schema = "integrations",
+            tables = arrayOf(
+                "ratko_push_content",
```

Comment:
Jätetäänkö muut ratko-taulut ja `lock` tarkoituksella putsaamatta, vai tuleeko ne jollain cascadella kun tuolta `ratko_push_content`-taulusta poistuu tavaraa (tai no, `lock` ei varmaan tulisi koskaan näin, mutta nuo muut?)

Reply:
Nämä on poimittu ottamalla nykyisistä käyttöpaikoista laajin. Sinällään noissa voisi ihan hyvin olla kaikkikin taulut mutta tuntui turhalta lähteä niitä lisäämään kun mikään testi ei niistä välitä. Toisaalta nimi kyllä lupaa että kaikki lähtisi että siinä mielessä voisin lisätäkin. Ja taas toisaalta, tauluja tulee varmasti jatkossa lisääkin -> olisi hyvä tottua ajatukseen että lisätään tuonne jos niillä on jollekin testille merkitystä.